### ISSUE TITLE
Book 05: Sentence 014

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#43]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/356c37f3-316a-44a0-a8d2-dfcb37198b12)

```
# sent_id = Book_05_-_The_Omens.txt:14
# text = No ioan, qujtetzammatia: in jqc aca ipan veuetzca, oactli: in juh conjtoa, vme vitz qujtoa: in quenman qualli, quenman amo qualli.
# text[norm] = No ihuan, quitetzammatia: in ihcuac aca ipan huehuetzca, huactli: in iuh conihtoa, ome huitz quihtoa: in quenman cualli, quenman amo cualli.
# text[orig] = No·ioan·,·qujtetzammatia·:·inj·qc¶aca·ipan·veuetzca·,·oactli·:·injuh·co¶njtoa·,·vme·vitz·qujtoa·:·inquenman¶qualli·,·quenman·amo·qualli·.
# text[eng] = Also with that, they thought it an omen: when at someone screeched, the huactli: that as they said, two ways it spoke: that sometimes good, sometimes not good.
# text[osp] = El segundo aguero que tenian: era quando oyan cantar, o charrear un aue, que llaman oactli, o oacton. Este aguero: era indiferente: que a las vezes pronunciaua bien, y a las vezes mal.
# text[osp_norm] = El segundo agüero que tenían: era cuando oían cantar, o charrear un ave, que llaman <huactli>, o <huacton>. Este agüero: era indiferente: que a las veces pronunciaba bien, y a las veces mal.
# references = A&D/162:2
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 34.48%
1	No	no	ADV	_	_	4	_	_	_
2	ioan	huan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	4	_	_	_
3	,	,	PUNCT	_	_	4	punct	_	_
4	qujtetzammatia	tetzammatia	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	_
5	:	:	PUNCT	_	_	10	punct	_	_
6	in	in	SCONJ	_	_	10	mark	_	_
7	jqc	ihcuac	ADV	_	_	10	advmod	_	_
8	aca	aca	DET|PRON	_	_	9	_	_	_
9	ipan	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	10	_	_	_
10	veuetzca	huehuetzca	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	4	_	_	_
11	,	,	PUNCT	_	_	12	punct	_	_
12	oactli	huactli	NOUN	_	State=Abs	10	nsubj	_	_
13	:	:	PUNCT	_	_	16	punct	_	_
14	in	in	SCONJ	_	_	16	mark	_	_
15	juh	iuh	ADV	_	_	16	_	_	_
16	conjtoa	ihtoa	VERB	_	Movement=And|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin4_	_	_	_	_
17	,	,	PUNCT	_	_	19	punct	_	_
18	vme	ome	NUM	_	_	19	_	_	_
19	vitz	huitz|huitza	NOUN	_	_	16	_	_	_
20	qujtoa	ihtoa	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	19	_	_	_
21	:	:	PUNCT	_	_	24	punct	_	_
22	in	in	DET|SCONJ	_	_	24	_	_	_
23	quenman	quenman	ADV	_	_	24	_	_	_
24	qualli	cualli	ADJ	_	_	19	_	_	_
25	,	,	PUNCT	_	_	28	punct	_	_
26	quenman	quenman	ADV	_	_	28	_	_	_
27	amo	amo	ADV	_	_	28	advmod	_	_
28	qualli	cualli	ADJ	_	_	24	_	_	_
29	.	.	PUNCT	_	_	4	punct	_	_
```

**Comment by ftyers** [comment]
López Austin has:
> Y también tenían por augurio cuando el huactli reía a carcajadas de alguno. Así le decían: viene a hablar en dos formas, a veces para bien, a veces para mal.

**Comment by Lguyogiro** [comment]
the _ome huitz quihtoa: quemman cualli, quemman amo cualli_ part is a bit weird. My understanding of this is "It came (as) two, they say: when it is god, when it is not good". For _ome huitz_, initially I thought maybe _ome_ would be the subj, but _huitz_ suggests a singular subject. so if this is some kind of secondary predication, would the syntax be like this?

![imagen](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/31f91068-df63-4a3b-ad42-dc3d27722fca)


It reminds me of the UD example _she entered the room sad_ which has `advcl(entered, sad)`, but this analysis still feels/looks weird.

**Comment by ftyers** [comment]
Oh, *huitz* is a `VERB` there? Hmm, there are a few similar examples:

```
...  auh in aquin ipan tlacatia, atle huel inezca, ome 	huitz	quihtoa.
... ihtoa, ca nel iuhqui itonal, in ce xochitl, ca ome 	huitz	quihtoa, cualli, ihuan acualli.
... teixti zan chico cualli, amo zan centettitica, ome 	huitz	quihtoa, in aquique ipan tlacatia in, ontei
... quenman amo itlacahui, cempopolivi, in quenman ome 	huitz	quihtoa, in azo cualli acanozomo cualli, in
... aca ipan huehuetzca, huactli: in iuh conihtoa, ome 	huitz	quihtoa: in quenman cualli, quenman amo cua
...  in acanozomo, ca itlahtol itech mocaqui: inic ome 	huitz	quihtoa, cualli, ihuan amo cualli.
...  ic contlahuantia: ihuan no mihtoa, in quenman ome 	huitz	quihtoa: no cualli quinextia, azo itla cual
```

I'm not sure about the secondary predication. If that is the case, what/where is the subject? According to the LA translation, it is:
> like this they said: it [the huactli] comes to speak in two [ways]: sometimes good, sometimes not good

But then the form of *huitz quihtoa* is weird, why is it not compounded, why is the *quihtoa* not in the future?

**Comment by Lguyogiro** [comment]
Yeah, I'm not super convinced about how we are handling that last part....specifically, under this analysis, what is the object of _quihtoa_?

> I wonder if there is some information that can be gleaned from the fact that conihtoa is in honorific and quihtoa is not.

I don't think Nahuatl of this era used _on-_ as a reverential, I believe it was exclusively directional (_conihtoa_, "say (it) thitherly"). Having said that, it may be a little weird for _huitz_ to be a non-parataxis daughter of _conihtoa_ (they say thitherly that it moves hitherly haha).


I'm thinking that the _in iuh conihtoa_ is actually modifying _huactli_, e.g. "huactli, as they (over there) call it".

Also, I found [another translation](https://www.academia.edu/81896926/Paleograf%C3%ADa_y_traducci%C3%B3n_del_n%C3%A1huatl_al_espa%C3%B1ol_del_Arte_adivinatoria_C%C3%B3dice_florentino_) of  _ome huitz quihtoa_ in a different context where they have it as I suggested above, namely "it comes (in) two (ways) they say". 

so what about this analysis of this last part.

![imagen](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/18ffdb15-b01e-47b3-9a6c-ba5bdad92878)

**Comment by ftyers** [comment]
How about this:

> And also they considered it an omen when at someone screeched, huactli, as they call it, two ways it comes they say, being sometimes good, sometimes not good.

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/13dd766b-7783-4fe3-a597-9156872a2db4)

The speech verb after the *huitz* should be parataxis according to the docs. The two questions I have:  
- Is there a subject of *quitetzammatia*? 
- Does `advcl` seem right for the last part? It starts with an *in* ...

**Comment by Lguyogiro** [comment]
LGTM

**Comment by ftyers** [comment]
Resolved in 1bb9483.
